### DISCUSSION TITLE
Supabase users

### DISCUSSION THREAD

**Discussion by sam-claire** [supabase/supabase#8551] [Questions]
I'm very new with supabase. I want to create my users without signing up them, we want that we will register them using our vuejs web app. Is this possible?

Thanks

**Comment by J0** [comment]
Hey,

Thanks for your query! I'm not sure I fully understood the question -- could you expand more on  how the sign up and registration flow will look like for your use case? If it's more comfortable for you, feel free to shoot us an email at support@supabase.io instead.

If it helps, one should be able to create a user but only verify the user's email  [via the admin endpoint](https://supabase.com/docs/reference/javascript/auth-api-createuser) at a later time. In this case the users would still be in the database at point of sign up though

Let us know!

**Comment by sam-claire** [comment]
so in other words, we can create users without signup? is that correct?
*Emond Sabiniano*

www.emondsoft-solutions.com

+639399393702
+639274007556
